extensions/intellij/intellij_generator_plugin/src/main/resources/templates/rx_bloc_with_extensions_with_default_states/rx_bloc_generated.dart.template

```diff
@@ -20,20 +20,20 @@ abstract class ${bloc_dollar_pascal_case}Bloc extends RxBlocBase
   final _compositeSubscription = CompositeSubscription();
 
   /// The state of [isLoading] implemented in [_mapToIsLoadingState]
-  Stream<bool> _isLoadingState;
+  Stream<bool>${bloc_use_null_safety} _isLoadingState;
 
   /// The state of [errors] implemented in [_mapToErrorsState]
-  Stream<String> _errorsState;
+  Stream<String>${bloc_use_null_safety} _errorsState;
 
   @override
-  Stream<bool> get isLoading => _isLoadingState ??= _mapToIsLoadingState();
+  Stream<bool>${bloc_use_null_safety} get isLoading => _isLoadingState ??= _mapToIsLoadingState();
```

Comment:
@nikolayIvanovPrime, could you please explain why the getter needs to be nullable?